vendor/install-getowner-polyfill.js

```diff
@@ -1,14 +1,15 @@
 import Ember from 'ember';
+import { lte } from 'ember-compatibility-helpers'
```

Comment:
I think we need to add `treeForVendor` or this won't work at all 🤔

Reply:
As a matter of fact, I don't fully understand how that `import Ember from 'ember'` there is working either (since we aren't transpiling this file at all it won't get replaced).

Reply:
Not sure I follow. This PR was orginally based on the ember-cli-babel PR which has the treeForVendor:

https://github.com/ember-polyfills/ember-getowner-polyfill/pull/34/files#diff-168726dbe96b3ce427e7fedce31bb0bc

I've since retargeted it onto master.

Reply:
Doh :facepalm: